site.someones.Lonewolf.yml

```diff
@@ -0,0 +1,62 @@
+app-id: site.someones.Lonewolf
```

Comment:
`http://someones.site/` doesn't seem to answer.

Reply:
Yeah, I know. I don't host anything there at the moment.

Ownership can be checked with:
```
nslookup -type=TXT lonewolf.someones.site 8.8.8.8
```
```
Server:         8.8.8.8
Address:        8.8.8.8#53

Non-authoritative answer:
Lonewolf.someones.site  text = "github.com/aichingm/lonewolf"
```

Reply:
is this an issue or can I mark this as resolved?

Reply:
you should really put a page there. It will be necessary to serve a file for verification purposes. (it's optional though)

Reply:
Ok, I think I will tackle this once I find time for the verification stuff.